latest_cov_build.py

```diff
@@ -94,3 +95,5 @@ def get_github_release_url(repo_slug):
 # Download Firefox coverage report
 codecoverage.download_coverage_artifacts(taskId, None)
 codecoverage.generate_report('tools/grcov', 'coveralls+', 'tests_report.json')
+
+subprocess.call(['hg', 'clone', 'https://hg.mozilla.org/mozilla-central/', 'firefox'])
```

Comment:
It would be better to use `robustcheckout` and `hglib`, like here: https://github.com/mozilla-releng/services/blob/master/src/shipit_code_coverage/shipit_code_coverage/codecov.py#L72.
robustcheckout ensures we're cloning as efficiently as possible.
However, since it's a one time thing, it's fine.
You should do two things though:
1) only clone if the repository doesn't exist yet,
2) checkout to the revision of the build (as we need the source code that was built at that time).